iotedgedev/cli.py

```diff
@@ -67,12 +67,18 @@ def docker():
                        "Use \".\" as NAME to create in the current folder.")
 @click.argument("name",
                 required=True)
+@click.option("--edge-runtime-version",
+              "-er",
+              default="1.0",
+              show_default=True,
+              required=False,
+              help="Specify the IoT Edge Runtime Version. Currently available 1.0 and 1.1")
 @add_module_options(envvars, init=True)
 @with_telemetry
-def new(name, module, template, group_id):
+def new(name, module, template, edge-runtime-version, group_id):
```

Comment:
>edge-runtime-version [](start = 32, length = 20)

I think we should allow only 1.0 or 1.1 for the runtime versions and throw an error if the value is not within that list.

Reply:
resolved in new commit https://github.com/Azure/iotedgedev/pull/466/commits/69998154b99df5ac7615e1f4d5b943ce6fb3c9cf

thanks for catching it!